reference/api/export.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Export
+description: "Migrate between instances with the `/export` route"
+---
+
+import { RouteHighlighter } from '/snippets/route_highlighter.mdx'
+
+import CodeSamplesExperimentalDeleteLogsStream1 from '/snippets/samples/code_samples_experimental_delete_logs_stream_1.mdx';
+
+Use the `/export` route to transfer data from your origin instace to a remote target instance. This is particularly useful when migrating from your local development environment to a Meilisearch Cloud instance.
+
+## Migrate database between instances
+
+<RouteHighlighter method="POST" path="/export" />
+
+Migrate data from the origin instance to a target instance.
+
+### Body
+
+| Name                | Type   | Default value | Description                                                                                               |
+| ------------------- | ------ | ------------- | --------------------------------------------------------------------------------------------------------- |
+| **`url`** *         | String | `null`        | The target instance's URL address. Required                                                               |
+| **`apiKey`** *      | String | `null`        | An API key with full admin access to the target instance                                                  |
+| **`payloadSize`** * | String | "50 MiB"      | A string specifying the payload size in a human-readable format                                           |
+| **`indexes`** *     | Object | `null`        | A set of patterns matching the indexes you want to export. Defaults to all indexes in the origin instance |
+
+#### `url`
+
+A string pointing to a remote Meilisearch instance, including its port if necessary.
+
+This field is required.
+
+#### `apiKey`
+
+A security key with full admin permissions to a secured Meilisearch instance.
+
+#### `payloadSize`
+
+The maximum size of each single data payload in a human-readable format such as `"100MiB"`. Larger payloads are generally more efficient, but require significantly more powerful machines.
+
+#### `indexes`
+
+A set of objects whose keys correspond to patterns matching the indexes you want to export. By default, Meilisearch exports all documents across all indexes.
+
+Each index object accepts the following fields:
+
+- `filter`: a [filter expression](/learn/filtering_and_sorting/filter_expression_reference) defining the subset of documents to export. Optional, defaults to `null`
+- `overrideSettings`: if `false`, uses the index settings as configured in the origin instance. if `true`, uses the target instance settings. Optional, defaults to `false`.
```

Comment:
```suggestion
- `overrideSettings`: if `false`, uses the index settings as configured in the target instance. if `true`, uses the origin instance settings. Optional, defaults to `false`.
```

Am I wrong, or is it a bad idea, and should it be the other way around? I mean in terms of API the default should be `true` 🤔

Reply:
Hey @guimachiavelli 👋
We found out with @Mubelotix that the parameter and default value are valid. The parameter defines that if the index already exists, we do not send the settings, and if it doesn't exist, we send them. If you set it to `true`, it will override the settings.